fe/fe-core/src/main/java/org/apache/doris/nereids/types/DataType.java

```diff
@@ -246,7 +246,16 @@ public static DataType convertPrimitiveFromStrings(List<String> types) {
                 dataType = DateV2Type.INSTANCE;
                 break;
             case "time":
-                dataType = TimeType.INSTANCE;
+                switch (types.size()) {
+                    case 1:
+                        dataType = TimeV2Type.INSTANCE;
+                        break;
+                    case 2:
+                        dataType = TimeV2Type.of(Integer.parseInt(types.get(1)));
```

Comment:
if u add scale on timev2 type, u should also update timev2 type related coersion just like datetimev2